instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -98,11 +98,15 @@ def analyze_query(query:, &block)
 
           def execute_query(query:, &block)
             attributes = {}
-            attributes['graphql.operation.name'] = query.selected_operation_name if query.selected_operation_name
-            attributes['graphql.operation.type'] = query.selected_operation.operation_type
+            operation_type = query.selected_operation.operation_type
+            operation_name = query.selected_operation_name
+
+            attributes['graphql.operation.name'] = operation_name if operation_name
+            attributes['graphql.operation.type'] = operation_type
             attributes['graphql.document'] = query.query_string
```

Comment:
Could you move these strings to constants? This will help us avoid an allocation every time a query is executed.